lib/preflight_check.js

```diff
@@ -4,7 +4,11 @@ const requiredEnvVars = {
   'SCSB_URL': process.env.SCSB_URL,
   'SCSB_API_KEY': process.env.SCSB_API_KEY,
   'ELASTICSEARCH_HOST': process.env.ELASTICSEARCH_HOST,
-  'RESOURCES_INDEX': process.env.RESOURCES_INDEX
+  'RESOURCES_INDEX': process.env.RESOURCES_INDEX,
+  'NYPL_API_BASE_URL': process.env.NYPL_API_BASE_URL,
+  'NYPL_OAUTH_URL': process.env.NYPL_OAUTH_URL,
+  'NYPL_OAUTH_ID': process.env.NYPL_OAUTH_ID,
+  'NYPL_OAUTH_SECRET': process.env.NYPL_OAUTH_SECRET
 }
```

Comment:
I think because we're short-circuiting logging when `NODE_ENV === 'test'`, I'm not getting the preflight_check error. Just fails quietly for me. I wonder if there's a way to continue logging [just] errors when running tests? Or cause preflight-check to fail more spectacularly (e.g. also log to console.error?) Or maybe I should have just read the docs.